src/components/Card/Card.test.tsx

```diff
@@ -1,26 +1,43 @@
 import * as React from 'react'
 import { shallow } from 'enzyme'
-import { CardContent, CardDescription, Progress } from 'semantic-ui-react'
+import { CardContent } from 'semantic-ui-react'
 
 import Card from './Card'
-import Spinner from '../common/Spinner'
-import AssigneesAvatar from '../AssigneesAvatar'
+import ICard from './../../models/Card'
+// import Spinner from '../common/Spinner'
+// import AssigneesAvatar from '../AssigneesAvatar'
 
-const cardType = {
-    title: 'Test',
-    labels: [],
-    percent: 40,
-    assignees: ['toto']
+const cardType: ICard = {
```

Comment:
Add in attribute only what will be returned by the '/card' route.  (tags, members, ... will not be here but in the reducers)

Reply:
I understand, but I have to change the interface CardProps in /components/Card/Card.tsx too, because in Card.test.tsx, this const cardType has the type ICard. OR in CardProps the attribute card is from a CardModel, which has not percent for exemple. 

What to you suggest ?